app.rb

```diff
@@ -1,9 +1,35 @@
 require 'sequel'
 require 'sinatra/base'
 require './lib/mac_formatter.rb'
+require './lib/session.rb'
+require './lib/user.rb'
 
 class App < Sinatra::Base
   get '/healthcheck' do
     'Healthy'
   end
+
+  get '/logging/post-auth/user/:username/mac/:mac/ap/:called_station_id/site/:site_ip_address/result/:authentication_result' do
+    if params.fetch(:authentication_result) == 'Access-Accept'
+      if params.fetch(:username) != 'HEALTH'
+        mac = MacFormatter.new.execute(mac: params.fetch(:mac))
+        Session.create(
+          username: params.fetch(:username),
+          mac: mac,
+          ap: params.fetch(:called_station_id),
+          siteIP: params.fetch(:site_ip_address),
+          building_identifier: params.fetch(:called_station_id),
+        )
+        user = User.find(username: params.fetch(:username))
+        user.last_login = Time.now.strftime('%y-%m-%d %H:%M:%S')
```

Comment:
What were your thoughts on removing the coupling between the user table and this service?

Reply:
Only reasonable option that comes to mind is to expose a update-last-login endpoint on the user-signup API.  We call that from here.

Reply:
Sounds sensible to me.  Feel as though we can introduce that as a separate slice in this iteration.

Reply:
Agreed